xarray/datatree_/docs/source/data-structures.rst

```diff
@@ -0,0 +1,197 @@
+.. currentmodule:: datatree
```

Comment:
this should be migrated to the xarray docs page: https://docs.xarray.dev/en/stable/user-guide/data-structures.html